primo/data_parser/well_data.py

```diff
@@ -864,12 +872,76 @@ def _process_input_data(self):
 
         LOGGER.info("Completed processing the essential inputs.")
 
+    def _get_state(self) -> str:
+        """
+        Attempts to infer the state for which the dataset belong.
+        Assumes that all data points belong to the same state and
+        we have at least one row of data
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        Two-digit code identifying the state for which the data belongs
+        """
+        wcn = self._col_names
+        lat = self.data[wcn.latitude].iloc[0]
+        long = self.data[wcn.longitude].iloc[0]
+        fips_code = get_fips_code(lat, long)
+        state = get_fips_part(fips_code, "STATE")
+        return state
+
+    def _convert_data_to_geodataframe(self) -> gpd.GeoDataFrame:
+        """
+        Returns a Geopandas GeoDataFrame object from well data
+        """
+        wcn = self._col_names
+        data = copy.deepcopy(self.data)
```

Comment:
I think creating a deepcopy is not needed. `gdf` is a new object created from data.

Reply:
you are right! Thanks for the call out :-)